docs: additional terms for dedicated instance (#2047)
caos/zitadel: docs/docs/legal/terms-of-service-dedicated.md

---
title: Dedicated Instance Terms
custom_edit_url: null
--- 
## General

Last revised: July 20, 2021

### Background

Within the scope of the Framework Agreement, the Customer may choose to purchase a subscription that requires a dedicated instance of ZITADEL. These additional terms for dedicated instance ("**Dedicated Instance Terms**") apply in addition to the Framework Agreement.

### Service

> CAOS operates and manages a **Dedicated Instance** of ZITADEL in a private infrastructure environment dedicated for Customer's use and provides support services for the customer according the Booking, these terms, agreed [**Service Level Description**](service-level-description), and [**Support Service Descriptions**](support-services).

Reviewer: ```suggestion
CAOS operates and manages a **Dedicated Instance** of ZITADEL in a private infrastructure environment dedicated for Customer's and provides support services for the Customer according the Booking, these terms, agreed [**Service Level Description**](service-level-description), and [**Support Service Descriptions**](support-services).
```

Author: Should be correct with the latest commit. Thanks.

> Each Dedicated Instance consists, except agreed otherwise in writing, a multi-zonal high-availability configuration that guarantees loads up to the specified [rate limits](https://docs.zitadel.ch/docs/legal/rate-limit-policy#what-rate-limits-do-apply).

Reviewer: ```suggestion
Each Dedicated Instance consists, except agreed otherwise in writing, of a multi-zonal high-availability configuration that guarantees loads up to the specified [rate limits](rate-limit-policy#what-rate-limits-do-apply).
```

### Operations

> CAOS will install and manage the Dedicated Cloud on virtual machines provided by CAOS through preferred infrastructure providers. Costs for infrastructure or cloud providers are not included in the Subscription, if not agreed otherwise in writing.

Reviewer: ```suggestion
CAOS will install and manage the Dedicated Instance on infracstructure provided by preferred cloud providers. Costs for infrastructure or cloud providers are not included in the Subscription, if not agreed otherwise in writing.
```

You may choose to provide the required infrastructure yourself. You must comply with the [requirements and prerequisites](https://docs.zitadel.ch/docs/guides/installation/managed-dedicated-instance) as documented.

> Please refer to the [annex](dedicated-instance-description) with regards to limitations of the deployment options, especially when using an own infrastructure provider. Accepting these terms you also accept the terms in the annex.

Reviewer: ```suggestion
Please refer to the [annex](dedicated-instance-annex) with regards to limitations of the deployment options, especially when using an own infrastructure provider. By accepting these terms you also accept the terms in the annex.
```

Reviewer: IMO this is the correct link.. btw. it breaks the docs build 
<img width="901" alt="Screenshot 2021-07-20 at 13 18 12" src="https://user-images.githubusercontent.com/9879976/126315384-0d128857-4149-4118-9033-b8d131b20174.png">

> You may not modify, maintain or attempt to modify the Dedicated Instance, except in with prior instructions by CAOS.

Reviewer: ```suggestion
You may not modify, maintain or attempt to modify the Dedicated Instance, except with prior instructions by CAOS.
```

CAOS will use the same backup strategy as for ZITADEL Cloud (public cloud) services, except otherwise agreed between you and CAOS in writing.

### Maintenance and Updates

We will access, modify, and maintain the Dedicated Instance at times solely determined by CAOS (**"Regular Maintenance"**).

> Under certain Subscription plans, the Customer may agree a custom frequency and times for modifications and updates. CAOS will coordinate the cadence and the modifications with the Customer. To guarantee the quality of service, maintenance will occur on regular basis, typically monthly or sooner for security or performance related patches (**"Emergency Maintenance"**), but no longer than on quarterly basis.

Reviewer: ```suggestion
Under certain subscription plans, the Customer may agree a custom frequency and times for changes and updates. CAOS will coordinate the cadence and the changes with the Customer. To guarantee the quality of service, maintenance will occur on regular basis, typically monthly or sooner for security or performance related patches (**"Emergency Maintenance"**), but no longer than on quarterly basis.
```

If you fail to permit CAOS to conduct Regular Maintenance for 3 consecutive months or Emergency Maintenance within 5 days of notification, then CAOS will raise this issue with the Customer via Escalation Process. In case the issue is not resolved 5 days after such an escalation, CAOS may terminate the subscription with 30 days prior written notice to Customer. CAOS is not obligated to provide the service according to the terms and SLA, nor is CAOS liable to any security breach or damages after failure to permit Regular Maintenance for 3 consecutive months, or Emergency Maintenance for 5 days after notification.

### Incidents

> Incidents are handled as documented in the [**Support Service Descriptions**](support-services). If the Customer choose in Booking to provide the required infrastructure, then any incidents related to the infrastructure of the Dedicated Instance have to be resolved through the Customer directly.

Reviewer: ```suggestion
Incidents are handled as documented in the [**Support Service Descriptions**](support-services). If the Customer choose in booking to provide the required infrastructure, then any incidents related to the infrastructure of the Dedicated Instance have to be resolved through the Customer directly.
```

Author: This is a definition. But we can use Purchase Order (added it to the ToS)

Reviewer: Yeah that might make it more clear

Author: Updated with the latest commit.
